app/handlers/upload.py

```diff
@@ -3,7 +3,7 @@
 import tornado.web
 import os
 import uuid
-from trafficcloud import api
+# from trafficcloud import api
```

Comment:
Remove changes to this file, as it will become obsolete when upload_routes is merged.